osu.Game/Rulesets/UI/ModIcon.cs

```diff
@@ -29,35 +30,17 @@ public float IconSize
 
         public new Color4 Colour
         {
-            get
-            {
-                return background.Colour;
-            }
-            set
-            {
-                background.Colour = value;
-            }
+            get { return background.Colour; }
+            set { background.Colour = value; }
         }
 
         public FontAwesome Icon
         {
-            get
-            {
-                return modIcon.Icon;
-            }
-            set
-            {
-                modIcon.Icon = value;
-            }
+            get { return modIcon.Icon; }
+            set { modIcon.Icon = value; }
         }
 
-        private void reapplySize()
-        {
-            background.TextSize = iconSize;
-            modIcon.TextSize = iconSize - 35;
-        }
-
-        public ModIcon()
+        public ModIcon(Mod m)

```

Comment:
make the variable called mod, not m. don't use single letter variables in method parameters.